docs/workflows.rst

```diff
@@ -359,27 +341,75 @@ Denoising
 =========
 :class:`~xcp_d.interfaces.nilearn.DenoiseNifti`, :class:`~xcp_d.interfaces.nilearn.DenoiseCifti`
 
-Temporal censoring [OPTIONAL]
+The denoising approach in XCP-D is heavily based on Nilearn's :footcite:p:`abraham2014machine`
+approach from :py:func:`~nilearn.signal.clean`,
+which was designed to follow recommendations made in :footcite:t:`lindquist2019modular`.
+
+Specifically, temporal filtering and confound regression are performed in separate stages,
+but the confounds are orthogonalized with respect to the temporal filter prior to confound
+regression,
+so that no variance from the temporal filter is reintroduced by the confound regression.
+
+XCP-D modifies Nilearn's approach in the following ways:
+
+1. XCP-D uses :func:`numpy.linalg.lstsq` to estimate betas instead of QR decomposition,
+   in order to denoise the interpolated data as well.
```

Comment:
Do we need to justify this choice?

Reply:
Justify the choice of `lstsq` over QR decomposition, or justify the decision to denoise the interpolated data?

Reply:
lstsq

Reply:
Gotcha. The reason we need to do it is because QR decomposition doesn't estimate betas, which we need to denoise the interpolated data. Should I basically just add that info to the documentation?

Reply:
Here is my updated text. WDYT?

```rst
XCP-D modifies Nilearn's approach in the following ways:

1. XCP-D uses :func:`numpy.linalg.lstsq` to estimate betas instead of QR decomposition,
   in order to denoise the interpolated data as well.
   -  QR decomposition will not produce betas that can be applied to the interpolated data.

2. XCP-D sets any leading or trailing high-motion volumes to the closest low-motion volume's values
   instead of extrapolating those volumes or removing them completely.

Both of these modifications allow XCP-D to produce a denoised, interpolated BOLD time series,
while Nilearn only produces the denoised, _censored_ BOLD time series.
The interpolated BOLD time series is necessary for DCAN-specific tools,
such as `biceps <https://biceps-cmdln.readthedocs.io/en/latest/>`_.
```